perftest/tests/sample/SamplePerfTest.cpp

```diff
@@ -0,0 +1,81 @@
+/* -*- Mode: C++; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4; fill-column: 100 -*- */
+/*
+ * Copyright the Collabora Online contributors.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#include <perftest/PerfTest.hpp>
+
+class SamplePerfTest : public MessagePerfTest
```

Comment:
Is it useful to derive your actual perf test suite from MessagePerfTest? It seemed to me that we're primarily interested in the number of CPU cycles the test uses, so CyclePerfTest would enough and you could drop MessagePerfTest from this PR.

For points 2) -> 6) from the checklist, I don't see a change with this update. Thanks.